src/generator/exercises/hello_world.cr

```diff
@@ -0,0 +1,34 @@
+require "./exercise_generator"
+require "./exercise_test_case"
+
+class HelloWorldGenerator < ExerciseGenerator
+  def exercise_name
+    "hello-world"
+  end
+
+  def test_cases
+    JSON.parse(data)["cases"].map do |test_case|
+      HelloWorldTestCase.from_json(test_case.to_json)
+    end
+  end
+end
+
+class HelloWorldTestCase < ExerciseTestCase
+  JSON.mapping(
+    description: String,
+    name: String | Nil,
+    expected: String
+  )
+
+  def workload
+    if !name
```

Comment:
I agree, but I have a rebuttal in the case when an else statement is used, I think unless can be confusing, but I could change this to `if name` and switch the logic around.